### DISCUSSION TITLE
getOne and filter:

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#1411] [Q&A] [answered]
Hello! in getOne, filter: 'somefilter' not working?

**Comment by ganigeorgiev** [comment, marked as answer]
`pb.collection('example').getOne('RECORD_ID')` works only with the record id and expects the client to satisfy the collection "View API" rule.

If you want to fetch a single item based on some custom criteria you can use the list/search methods, for example `getFirstListItem`:

```js
const record = await pb.collection('example').getFirstListItem('title ~ "test"')
```

**Reply by pgswow** [reply]
Yes, thanks, that's exactly what I use, I just wanted to clarify about getOne =)
